readthedocs/api/base.py

```diff
@@ -32,7 +32,7 @@
 class ProjectResource(ModelResource, SearchMixin):
     users = fields.ToManyField('readthedocs.api.base.UserResource', 'users')
 
-    class Meta:
+    class Meta(object):
```

Comment:
The linter complains if python old-style classes are used. Django makes the `Meta` classes into new style classes automatically since this ticket: https://code.djangoproject.com/ticket/7342

The restframework however does not. So that's why we made them new-styles classes here by hand. I think of it as "future-proofing" as all classes will be new style classes in Python 3, so we do some testing in that regard here if we make the switch now.